pages/_app.tsx

```diff
@@ -138,7 +130,7 @@ function MyApp({ Component, pageProps }: AppProps) {
           <ConnectionProvider endpoint={endpoint}>
             <WalletProviderSolana wallets={wallets} autoConnect>
               <WalletModalProvider>
-                <WalletProvider>
+                <WalletProviderDeprecated>
```

Comment:
I don't want to further complicate our stack with 2 graphql clients. Lets just use our graph endpoint for now but will make the switch as hasura for the gateway when we get it all tuned up. At that point we just switch the endpoint.